src/sql/src/func.rs

```diff
@@ -1975,6 +2034,21 @@ lazy_static! {
         }
     };
 
+    pub static ref INFORMATION_SCHEMA_BUILTINS: HashMap<&'static str, Func> = {
+        use ParamType::*;
+        builtins! {
+            "_pg_expandarray" => Set {
+                params!(ArrayAny) => sql_impl_table_func("
+                    SELECT $1[s] AS x, s - pg_catalog.array_lower($1,1) + 1 AS n
+                    FROM pg_catalog.generate_series(
+                        pg_catalog.array_lower($1,1),
```

Comment:
I think you are correct. It is this way because that's how it is in postgres (https://github.com/postgres/postgres/blob/16e3ad5d143795b05a21dc887c2ab384cce4bcb8/src/backend/catalog/information_schema.sql#L43) and I suppose it'll also be future-proof if we ever support the weirder array syntax.

Reply:
Cool! Just wanted to make sure my thinking was clear.